packages/node_modules/@webex/plugin-meetings/src/scheduled-meetings/index.js

```diff
@@ -0,0 +1,104 @@
+import {StatelessSparkPlugin} from '@ciscospark/spark-core';
+
+import {MEETINGS, EVENT_TRIGGERS, CALENDAR_EVENTS} from '../constants';
+
+import ScheduledMeetingsRequest from './request';
+import ScheduledMeetingsCollection from './collection';
+
+/**
+ * @class ScheduledMeetings
+ */
+export default class ScheduledMeetings extends StatelessSparkPlugin {
+  namespace = MEETINGS;
+
+  /**
+   *
+   * @param {Object} attrs
+   * @param {Object} options
+   */
+  constructor(attrs, options) {
+    super({}, options);
+    this.scheduledMeetingsRequest = new ScheduledMeetingsRequest({}, options);
+  }
+
+  /**
+   * TODO: when TTL or otherwise is done, add these in the appropriate spot in the ScheduledMeetingsCollection
+   * registers mercury events and sends out calendar events to listeners
+   * wrapper around the mercury calendar events
+   * @returns {undefined}
+   */
+  registerCalendarEvents() {
+    this.spark.internal.mercury.on(CALENDAR_EVENTS.CREATE, (envelope) => {
+      this.trigger(EVENT_TRIGGERS.CALENDAR_CREATE, {
+        data: envelope.data,
+        type: CALENDAR_EVENTS.CREATE
+      });
+    });
+    this.spark.internal.mercury.on(CALENDAR_EVENTS.UPDATE, (envelope) => {
+      this.trigger(EVENT_TRIGGERS.CALENDAR_UPDATE, {
+        data: envelope.data,
+        type: CALENDAR_EVENTS.UPDATE_MINIMAL
+      });
+    });
+    this.spark.internal.mercury.on(CALENDAR_EVENTS.CREATE_MINIMAL, (envelope) => {
+      this.trigger(EVENT_TRIGGERS.CALENDAR_CREATE, {
+        data: envelope.data,
+        type: CALENDAR_EVENTS.CREATE_MINIMAL
+      });
+    });
+    this.spark.internal.mercury.on(CALENDAR_EVENTS.UPDATE_MINIMAL, (envelope) => {
+      this.trigger(EVENT_TRIGGERS.CALENDAR_UPDATE, {
+        data: envelope.data,
+        type: CALENDAR_EVENTS.UPDATE_MINIMAL
+      });
+    });
+    this.spark.internal.mercury.on(CALENDAR_EVENTS.DELETE, (envelope) => {
+      this.trigger(EVENT_TRIGGERS.CALENDAR_DELETE, {
+        data: envelope.data,
+        type: CALENDAR_EVENTS.DELETE
+      });
+    });
```

Comment:
It looks like we are just doing pass throughs of all these events. Could the end user subscribe to the same thing on mercury themselves? 

I guess I see the benefit of the flow/documentation not referencing a separate internal plugin is nice.